### PR TITLE
Improved the UI of login page

### LINKED ISSUE #1064: Improve the UI of the login page
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [X] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
The current UI of login page is good but it can be improved.
screen shot 2018-03-15 at 9 06 29 pm

**Expected behavior:**
Instead of having the login using social accounts on the side we should have it in the login box itself.

**Would you like to work on the issue?**

YES

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Improved the UI of the login page
Before:
screen shot 2018-03-15 at 9 06 29 pm
After:
screen shot 2018-03-16 at 9 18 30 pm

Fixes #1064

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-1068-open-event-frontend-fossasia.surge.sh

File: app/templates/components/forms/login-form.hbs [hunk 1]
```diff
@@ -34,32 +34,33 @@
             </button>
             <a href="{{href-to 'reset-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
           </div>
+          <hr width="45%" align="left" style="width:45%;float:left;"><span style="display:inline-block;width:10%;text-align:center;">OR</span><hr width="45%" align="right" style="width:45%;float:right;">
```
Comment (niranjan94) [inline app/templates/components/forms/login-form.hbs]: Use Semantic UI's divider instead
https://semantic-ui.com/elements/divider.html

**Comment by niranjan94** [comment]
@harsh-jindal any update here ?

**Comment by harsh-jindal** [comment]
@niranjan94 I'm almost done with this, will update shortly.

**Comment by agarwalprashasti** [comment]
sir is this issue still open??

**Comment by agarwalprashasti** [comment]
is this issue open??

**Comment by niranjan94** [comment]
Hi @agarwalprashasti . This is a PR. Not an issue. Waiting for the OP @harsh-jindal to give us an update :) 

Hi @harsh-jindal ... any progress on this front ?

**Comment by CosmicCoder96** [comment]
closing due to inactivity.

**Pull request closed without merging**